frontend/src/components/data-table/data-table.tsx

```diff
@@ -165,7 +263,11 @@ const DataTableInternal = <TData,>({
           />
         )}
         <Table>
-          <TableHeader>{renderHeader()}</TableHeader>
+          <TableHeader>
```

Comment:
can you update (a few lines above

`<div className={cn(className || "rounded-md border")}>`
to 
`<div className={cn(className || 'rounded-md border overflow-hidden')}>`

otherwise the white background doesn't get clipped in the corners